### ISSUE TITLE
SVO Extraction & Visualization Pathway Error

### ISSUE THREAD

**Issue by spark1100817** [NLP-Suite/NLP-Suite#1249]
SVO extraction & visualization features failing due to pathway name (saving new coref files with backslash instead of "/").
Using a Windows machine with version 4.2.8.

![image](https://github.com/NLP-Suite/NLP-Suite/assets/143023864/d8b56a74-7334-46bf-a005-f12601a1be63)
![image](https://github.com/NLP-Suite/NLP-Suite/assets/143023864/7081f646-3206-4297-85e5-352aee5793f9)

**Comment by bianshuyang** [comment]
I do not believe you are on the right version. We are already at 4.3.3 but not 4.2.8

**Comment by bianshuyang** [comment]
Potentially replace all files in /src with the newest and try again: https://github.com/NLP-Suite/NLP-Suite/tree/current-stable/src

**Comment by bianshuyang** [comment]
But the release version is also off... potentially click "CLOSE" on the right lower corner to close and update NLP Suite, observe carefully what update message it shows... it seems auto update is not there

**Comment by spark1100817** [comment]
Hi Simon, thank you for the quick response! I have since updated to the latest release since this was posted, but I am still getting the same error:
![image](https://github.com/NLP-Suite/NLP-Suite/assets/143023864/3511278e-dee1-4657-8342-09577dfad932)
I also tried replacing the src directory.

**Comment by bianshuyang** [comment]
Thanks. I have found: https://stackoverflow.com/questions/16333569/mixed-slashes-with-os-path-join-on-windows That means this is not your problem any more. We need to improve the code a bit with @Emory-NLP . THere are >=106 instances of os path join and it's a hefty task. 
Has any of the other errors occur? Is this a widespread problem for every widget? Further, could you, in the meantime, inform us of the Windows version (10 / 11, etc?)?  That would speed up the development.

**Comment by spark1100817** [comment]
So far, I have only noticed this issue with the SVO Extractor and Visualization and Vocabulary Analysis under the Style Analysis GUI. I am using Windows 11.

Let me know if there is any other information I can provide - thank you again!
